modules/fuzzy/samples/filtering.cpp

```diff
@@ -0,0 +1,30 @@
+#include "opencv2/opencv.hpp"
```

Comment:
When you build contrib modules together with the main repo all samples will be built under the same sample directory. Therefore I suggest you to rename the samples to for example fuzzy_filtering.cpp, to make a difference with the current filtering techniques.